internal/rukpak/convert/registryv1_test.go

```diff
@@ -24,469 +23,537 @@ import (
-				Expect(err).To(HaveOccurred())
-				Expect(plainBundle).To(BeNil())
-			})
-
-			It("should error when single namespace mode is disabled with more than one target namespaces", func() {
-				csv := baseCSV.DeepCopy()
-				csv.Spec.InstallModes = []v1alpha1.InstallMode{{Type: v1alpha1.InstallModeTypeSingleNamespace, Supported: false}}
-
-				By("creating a registry v1 bundle")
-				watchNamespaces = []string{"testWatchNs1", "testWatchNs2"}
-				unstructuredSvc := convertToUnstructured(svc)
-				registryv1Bundle = RegistryV1{
-					PackageName: "testPkg",
-					CSV:         *csv,
-					Others:      []unstructured.Unstructured{unstructuredSvc},
-				}
-
-				By("converting to plain")
-				plainBundle, err := Convert(registryv1Bundle, installNamespace, watchNamespaces)
-				Expect(err).To(HaveOccurred())
-				Expect(plainBundle).To(BeNil())
-			})
-
-			It("should error when all namespace mode is disabled with target namespace containing an empty string", func() {
-				csv := baseCSV.DeepCopy()
-				csv.Spec.InstallModes = []v1alpha1.InstallMode{
-					{Type: v1alpha1.InstallModeTypeAllNamespaces, Supported: false},
-					{Type: v1alpha1.InstallModeTypeOwnNamespace, Supported: true},
-					{Type: v1alpha1.InstallModeTypeSingleNamespace, Supported: true},
-					{Type: v1alpha1.InstallModeTypeMultiNamespace, Supported: true},
-				}
-
-				By("creating a registry v1 bundle")
-				watchNamespaces = []string{""}
-				unstructuredSvc := convertToUnstructured(svc)
-				registryv1Bundle = RegistryV1{
-					PackageName: "testPkg",
-					CSV:         *csv,
-					Others:      []unstructured.Unstructured{unstructuredSvc},
-				}
-
-				By("converting to plain")
-				plainBundle, err := Convert(registryv1Bundle, installNamespace, watchNamespaces)
-				Expect(err).To(HaveOccurred())
-				Expect(plainBundle).To(BeNil())
-			})
-
-			It("should propagate csv annotations to chart metadata annotation", func() {
-				csv := baseCSV.DeepCopy()
-				csv.Spec.InstallModes = []v1alpha1.InstallMode{{Type: v1alpha1.InstallModeTypeMultiNamespace, Supported: true}}
-
-				By("creating a registry v1 bundle")
-				watchNamespaces = []string{"testWatchNs1", "testWatchNs2"}
-				unstructuredSvc := convertToUnstructured(svc)
-				registryv1Bundle = RegistryV1{
-					PackageName: "testPkg",
-					CSV:         *csv,
-					Others:      []unstructured.Unstructured{unstructuredSvc},
-				}
-
-				By("converting to helm")
-				chrt, err := toChart(registryv1Bundle, installNamespace, watchNamespaces)
-				Expect(err).NotTo(HaveOccurred())
-				Expect(chrt.Metadata.Annotations["olm.properties"]).NotTo(BeNil())
-			})
-		})
-
-		Context("Should read the registry+v1 bundle filesystem correctly", func() {
-			It("should include metadata/properties.yaml and csv.metadata.annotations['olm.properties'] in chart metadata", func() {
-				fsys := os.DirFS("testdata/combine-properties-bundle")
-				chrt, err := RegistryV1ToHelmChart(context.Background(), fsys, "", nil)
```

Comment:
This is the function that actually implements the propagation of metadata/properties.yaml into the CSV. And I don't think I see this function called in the new test set.